imapserver/fetch.go

```diff
@@ -75,6 +75,21 @@ func (c *Conn) handleFetch(dec *imapwire.Decoder, numKind NumKind) error {
 		}
 	}
 
+	if dec.SP() && dec.Special('(') {
+		var param string
+		if !dec.ExpectAtom(&param) {
+			return dec.Err()
+		}
+
+		if strings.ToUpper(param) == "CHANGEDSINCE" {
+			if !dec.ExpectSP() || !dec.ExpectModSeq(&options.ChangedSince) || !dec.ExpectSpecial(')') {
+				return dec.Err()
+			}
```

Comment:
To ease server implementations, could we set `options.ModSeq = true` here as well? From the RFC:

> When the CHANGEDSINCE FETCH modifier is specified, it implicitly
      adds the MODSEQ FETCH message data item